quick-start/README.md

```diff
@@ -57,6 +56,34 @@ Confirm the plugin is successfully installed by
 
 `$ docker plugin ls`
 
+
+### HPE 3PAR Fibre Channel plugin
+
+Support for HPE 3PAR FC Volume Plugin has been added in hpestorage/hpedockervolumeplugin:2.0 and it is tested against Docker 17.06 EE.
+
+Execute below commands to install the FC plugin:
+
+```
+# Please follow the pre-requisites and other details from the previous instructions on how to install plugin on different platforms.
+
+$ docker plugin install store/hpestorage/hpedockervolumeplugin:2.0 –-disable –-alias hpe 
+$ docker plugin set hpe glibc_libs.source=/lib64 certs.source=/tmp
```

Comment:
above glibc_libs.source=/lib64 is applicable for CentOS/RHEL and not for Ubuntu, you can change this line accordingly.